server/applicationPattern/applicationPattern/onfModel/services/models/logicalTerminationPoint/ConfigurationInputV2.js

```diff
@@ -1,4 +1,5 @@
 /**
+ * @deprecated please use ConfigurationInputV2.js
```

Comment:
this line shall be added to ConfigurationInput.js file instead of ConfigurationInputV2.js file

Reply:
Modified it accordingly. Thanks.